internal/trainer/service/application.go

```diff
@@ -13,10 +13,10 @@ import (
 )
 
 func NewApplication(ctx context.Context) app.Application {
-	firestoreClient, err := firestore.NewClient(ctx, os.Getenv("GCP_PROJECT"))
-	if err != nil {
-		panic(err)
-	}
+	firestoreClient := &firestore.Client{}
```

Comment:
@roblaszczak, I am wondering how to initialize the Firestore client when generating C4 diagrams. I have a couple of ideas:
1. Generate diagrams in docker where Firestore emulator is available
2. Do simple DI with an auxiliary mocked configuration

What would you recommend?

Reply:
ad 1. it doesn't need to be generated **in** Docker, but **with** Docker, right? I guess that passing envs and running docker with Firestore is fine and should work pretty out of the box :wink:

Reply:
Apparently, after I faked a couple of env vars it worked without docker-compose running